Generators/AvailableElementsGenerator.cs

```diff
@@ -0,0 +1,269 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Svg.Generators
+{
+    [Generator]
+    public class AvailableElementsGenerator : ISourceGenerator
+    {
+        private const string AttributeText = @"// <auto-generated />
+#nullable disable
+using System;
+
+namespace Svg
+{
+    [AttributeUsage(AttributeTargets.Class, Inherited = false, AllowMultiple = false)]
+    public sealed class ElementFactoryAttribute : Attribute
+    {
+    }
+}";
+
+        public void Initialize(GeneratorInitializationContext context)
+        {
+            // System.Diagnostics.Debugger.Launch();
```

Comment:
can be removed

Reply:
Necessary for Source Generators debugging. When you uncomment during build debugger will be attached, this is recommend way to debug.